components/base/kinematicbase/ptgKinematics.go

```diff
@@ -111,6 +112,9 @@ func (ptgk *ptgBaseKinematics) GoToInputs(ctx context.Context, inputs []referenc
 		return errors.New("inputs to ptg kinematic base must be length 3")
 	}
 
+	logger := golog.NewLogger("ddk")
```

Comment:
I don't think we want to be defining a brand new logger with every GoToInputs call

Reply:
this was something I did for testing convenience, will fix